agent_deploy/kubernetes/sysdig-agent-clusterrole.yaml

```diff
@@ -96,6 +96,14 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - policy
+  resources:
+  - poddisruptionbudgets
```

Comment:
How much cpu/mem impact does this have on the agent? I assume these don't change often, so possibly mostly  a small memory cost. Is that right?